### PR TITLE
feat: add users for writing only

### PR DESCRIPTION
**Pull request by OlleLarsson**

### CODE REVIEW

File: charts/influxdb/templates/elastisys_custom/influx-auth-secret.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ metadata:
     heritage: "{{ .Release.Service }}"
     release: "{{ .Release.Name }}"
   name: {{ template "influxdb.fullname" . }}-auth
-data:
-  INFLUXDB_USER: {{ .Values.elastisys_custom.influxdb_user | b64enc }}
-  INFLUXDB_PWD: {{ .Values.elastisys_custom.influxdb_password | b64enc }}
+stringData:
+  INFLUXDB_ADMIN_USER: {{ .Values.elastisys_custom.influxdbAdminUser }}
+  INFLUXDB_ADMIN_PWD: {{ .Values.elastisys_custom.influxdbAdminPassword }}
```
Comment (lentzi90) [inline charts/influxdb/templates/elastisys_custom/influx-auth-secret.yaml:12]: Does this name change affect anything else?

**Review by lentzi90 — COMMENTED** [review]
This looks fine!
Just wondering about the name change?

File: charts/influxdb/templates/elastisys_custom/influx-auth-secret.yaml (hunk 1 above)
Comment (OlleLarsson) [reply charts/influxdb/templates/elastisys_custom/influx-auth-secret.yaml:12]: Yes, it effects https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/influxdb.yaml.gotmpl#L46-L58
The keys will need to be updated to match the new names :)

File: charts/influxdb/values.yaml [hunk 2]
```diff
@@ -1,9 +1,12 @@
 elastisys_custom:
   ## The following values must be set in order to use the chart.
-  # influx_addr:
```
Comment (viktor-f) [inline charts/influxdb/values.yaml:3]: Is it intentional to remove this line?

**Review by viktor-f — COMMENTED** [review]
You should probably up the chart version as well.

Also how does influx create these new users? Do we have to add some `CREATE USER ...` here: https://github.com/elastisys/compliantkubernetes-apps/blob/71cb5e75dc26e7235caf6f1eb3bf9a497ff57cc2/helmfile/values/influxdb.yaml.gotmpl#L75 ?

File: charts/influxdb/values.yaml (hunk 2 above)
Comment (OlleLarsson) [reply charts/influxdb/values.yaml:3]: Yes :)
It's not used https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/influxdb.yaml.gotmpl#L1-L4

**Comment by OlleLarsson** [comment]
> You should probably up the chart version as well.

Will do

> Also how does influx create these new users? Do we have to add some `CREATE USER ...` here: https://github.com/elastisys/compliantkubernetes-apps/blob/71cb5e75dc26e7235caf6f1eb3bf9a497ff57cc2/helmfile/values/influxdb.yaml.gotmpl#L75 ?

Yes correct.

**Review by lentzi90 — APPROVED** [review]

**Review by viktor-f — APPROVED** [review]

**Pull request merged**